### DISCUSSION TITLE
Disabling some components in spacy

### DISCUSSION THREAD

**Discussion by kaushalkrishna2000** [SamEdwardes/spaCyTextBlob#23] [Q&A]
Hi,

I want to know if disabling any component of default spacy pipeline will affect the sentiment analysis after attaching spacytextblob ?

Like for lemmatizing we can remove parser and ner . 

I want to know if we can do the same in sentiment analysis using spacytextblob ?

Thank you

**Comment by SamEdwardes** [comment]
You should be able to disable any spacy component and and spacytextblob will still work. This is because to create the TextBlob object is created from `doc.text`.

https://github.com/SamEdwardes/spacytextblob/blob/9a87f3d81add90898fef8f32056e3353c6c1e82b/spacytextblob/spacytextblob.py#L59-L66.

Please let me know if anything does break when you disable default pipelines!
